### ISSUE TITLE
trouble using this

### ISSUE THREAD

**Issue by cptnbananajam** [rinnocenti/innocenti-openlock#12]
I followed the instruction on a few videos, and your post here. I have actors set up and items created. I can place a lootsheet transparent token, which can then be given a lock. When the players use the open lock macro or find trap, it say not a valid loot. Am I missing something?

**Comment by cptnbananajam** [comment]
I read in an older post to check f12 when doing this command. It appeared to come from the furnace. The error is:
Uncaught (in promise) TypeError: lootToken.actor.items.getFlag is not a function. 

Is this an issue with the furnace and not the open lock?

**Comment by rinnocenti** [comment]
Uodate the module..

**Comment by cptnbananajam** [comment]
I updated all of my assets and made sure everything was up to date. I'm still getting the same messages. I'm doing this on forge-vtt.com. does that make a difference?

**Comment by cptnbananajam** [comment]
The module version is 0.3.51, and I refreshed the server. The loot macro seemed to work, saying the token was looted. After the first time of it saying that, it gave me the same issue. The token only had a lock on it and the lock wasn't looted from the sheet. It tried to loot the currency that was there, but there wasn't anything but a lock on it. So I am not sure what I am doing wrong. The lock shares the wall coordinates with the door, values are set, and I clicked the enable button. It still tells me that it is not a valid loot. I even reset the tokens I created. I went back to the instructions, deleted the tokens I made and remade them following the instructions.
